p4include/v1model.p4

```diff
@@ -134,7 +241,35 @@ enum HashAlgorithm {
     xor16
 }
 
+/***
+ * mark_to_drop() is a primitive action that modifies
+ * standard_metadata.egress_spec to an implementation-specific special
+ * value that in some cases causes the packet to be dropped at the end
+ * of ingress or egress processing.
+ *
+ * See
+ * https://github.com/p4lang/behavioral-model/blob/master/docs/simple_switch.md
+ * -- in particular the section "Pseudocode for what happens at the
+ * end of ingress and egress processing" -- for the relative priority
+ * of the different possible things that can happen to a packet when
+ * ingress and egress processing are complete.
+ */
 extern void mark_to_drop();
+
+/***
+ * Calculate a hash function of the value specified by the data
+ * parameter.  Call this value H.  The value written to the result out
+ * parameter is then: (base + (H % max)).
+ *
+ * TBD: Do p4c and behavioral-model support run-time variable values
```

Comment:
For the bmv2 implementation specifically, I expect that simple_switch will segfault for a `max` of 0 because it will attempt to do a modulo 0 operation. Just like for `random` we can add a check and a log message.

Reply:
I will add another commit in a moment that proposes the behavior that if max=0, then the value written to the out parameter will always be equal to base, the same behavior as if max=1.  A warning in the log seems reasonable, but if explicitly documented maybe not high priority, since people may even come to rely upon such documented behavior.

If you prefer the documentation to say that the value written to the out parameter is unspecified if max=0, that also seems reasonable to me.

The current bmv2 implementation does segfault, as you suspected, according to my testing.